weblate/templates/dictionary.html

```diff
@@ -31,7 +31,7 @@
     </a>
     <ul class="dropdown-menu">
 <li><a href="{% url 'download_dictionary' project=project.slug lang=language.code %}?format=csv">{% trans "Comma separated values (CSV)" %}</a></li>
-<li><a href="{% url 'download_dictionary' project=project.slug lang=language.code %}?format=po">{% trans "Gettext (PO)" %}</a></li>
+<li><a href="{% url 'download_dictionary' project=project.slug lang=language.code %}?format=po">{% trans "gettext (PO)" %}</a></li>
```

Comment:
Same as above - "Comma separated" is with capital letter, same should be done for Gettext.

Reply:
gettext supports MO and other things too, no?